### PR TITLE
clean up typescript examples

### PR DESCRIPTION
**Pull request by megsdevs**
TODO handle RFQ service using Node stream

### CODE REVIEW

File: docs/examples/typescript/src/RFQ_maker.ts [hunk 1]
```diff
@@ -106,127 +136,150 @@ async function respondToRfqs() {
...
+      const optionTokenContract = new Contract(
+        VALOREM_CLEAR_ADDRESS,
+        IERC1155,
+        provider
       );
 
-    };
-  };
-};
+      let txReceipt = await (
```
Comment (dalechyn) [inline docs/examples/typescript/src/RFQ_maker.ts]: Use const here

File: docs/examples/typescript/src/RFQ_maker.ts [hunk 2]
```diff
@@ -106,127 +136,150 @@ async function respondToRfqs() {
...
-
-const clearinghouseContract = new Contract(VALOREM_CLEAR_ADDRESS, IValoremOptionsClearinghouse, provider);
+const clearinghouseContract = new Contract(
+  VALOREM_CLEAR_ADDRESS,
+  IValoremOptionsClearinghouse,
+  provider
+);
 const wethContract = new Contract(WETH_ADDRESS, IERC20, provider);
 
+import ISeaport from '../../abi/ISeaport.json';
```
Comment (dalechyn) [inline docs/examples/typescript/src/RFQ_maker.ts]: Move to top

**Review by dalechyn — CHANGES_REQUESTED** [review]
Few ones

File: examples/typescript/gen/valorem/trade/v1/auth_connect.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+import { Empty, NonceText, VerifyText } from "./auth_pb";
+import { MethodKind } from "@bufbuild/protobuf";
+import { H160 } from "./types_pb";
+
+/**
+ * @generated from service quay.Auth
+ */
+export const Auth = {
+  typeName: "quay.Auth",
```
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/auth_connect.ts]: trade.Auth

File: examples/typescript/gen/valorem/trade/v1/auth_pb.ts [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+   */
+  nonce = "";
+
+  constructor(data?: PartialMessage<NonceText>) {
+    super();
+    proto3.util.initPartial(data, this);
+  }
+
+  static readonly runtime: typeof proto3 = proto3;
+  static readonly typeName = "quay.NonceText";
```
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/auth_pb.ts]: trade

File: examples/typescript/gen/valorem/trade/v1/auth_pb.ts [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+   */
+  body = "";
+
+  constructor(data?: PartialMessage<VerifyText>) {
+    super();
+    proto3.util.initPartial(data, this);
+  }
+
+  static readonly runtime: typeof proto3 = proto3;
+  static readonly typeName = "quay.VerifyText";
```
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/auth_pb.ts]: trade

File: examples/typescript/gen/valorem/trade/v1/rfq_connect.ts [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+// @ts-nocheck
+
+import { QuoteRequest, QuoteResponse } from "./rfq_pb";
+import { MethodKind } from "@bufbuild/protobuf";
+
+/**
+ * @generated from service quay.RFQ
+ */
+export const RFQ = {
+  typeName: "quay.RFQ",
```
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/rfq_connect.ts]: trade

File: examples/typescript/gen/valorem/trade/v1/rfq_pb.ts [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+   */
+  SELL = 1,
+
+  /**
+   * @generated from enum value: INVALID = 255;
+   */
+  INVALID = 255,
+}
+// Retrieve enum metadata with: proto3.getEnumType(Action)
+proto3.util.setEnumType(Action, "quay.Action", [
```
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/rfq_pb.ts]: trade

File: examples/typescript/gen/valorem/trade/v1/types_pb.ts [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+   */
+  lo = 0;
+
+  constructor(data?: PartialMessage<H96>) {
+    super();
+    proto3.util.initPartial(data, this);
+  }
+
+  static readonly runtime: typeof proto3 = proto3;
+  static readonly typeName = "quay.H96";
```
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/types_pb.ts]: trade

File: examples/typescript/gen/valorem/trade/v1/types_pb.ts
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/types_pb.ts:1]: This generated file should not be in version control

File: examples/typescript/gen/valorem/trade/v1/seaport_pb.ts
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/seaport_pb.ts:1]: This generated file should not be in version control

File: examples/typescript/gen/valorem/trade/v1/rfq_pb.ts
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/rfq_pb.ts:1]: This generated file should not be in version control

File: examples/typescript/gen/valorem/trade/v1/rfq_connect.ts
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/rfq_connect.ts:1]: This generated file should not be in version control

File: examples/typescript/gen/valorem/trade/v1/auth_pb.ts
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/auth_pb.ts:1]: This generated file should not be in version control

File: examples/typescript/gen/valorem/trade/v1/auth_connect.ts
Comment (0xAlcibiades) [inline examples/typescript/gen/valorem/trade/v1/auth_connect.ts:1]: This generated file should not be in version control

File: examples/typescript/src/RFQ_maker.ts
Comment (0xAlcibiades) [inline examples/typescript/src/RFQ_maker.ts:1]: Looks good.

File: examples/typescript/src/RFQ_taker.ts
Comment (0xAlcibiades) [inline examples/typescript/src/RFQ_taker.ts:1]: Looks good.

**Review by 0xAlcibiades — CHANGES_REQUESTED** [review]
Feedback has been addressed.

**Review by 0xAlcibiades — APPROVED** [review]

**Pull request merged**